modules/manually-configure-iam-nutanix.adoc

```diff
@@ -0,0 +1,140 @@
+--to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
+quay.io/<path_to>/ocp-release:<version>
+----
++
+<1> Specify the path to the directory that contains the files for the component `CredentialsRequests` objects. If the specified directory does not exist, this command creates it.
++
+.Sample `CredentialsRequest` object
+[source,yaml]
+----
+  apiVersion: cloudcredential.openshift.io/v1
+  kind: CredentialsRequest
+  metadata:
+    annotations:
+      include.release.openshift.io/self-managed-high-availability: "true"
+    labels:
+      controller-tools.k8s.io: "1.0"
+    name: openshift-machine-api-nutanix
+    namespace: openshift-cloud-credential-operator
+  spec:
+    providerSpec:
+      apiVersion: cloudcredential.openshift.io/v1
+      kind: NutanixProviderSpec
+    secretRef:
+      name: nutanix-credentials
+      namespace: openshift-machine-api
+----
+
+. Use the `ccoctl` tool to process all of the `CredentialsRequest` objects in the `credrequests` directory by running the following command:
++
+[source,terminal]
+----
+$ ccoctl nutanix create-shared-secrets \
+--credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
+--output-dir=<ccoctl_output_dir> \ <2>
+--credentials-source-filepath=<path_to_credentials_file> <3>
+----
++
+<1> Specify the path to the directory that contains the files for the component `CredentialsRequests` objects.
+<2> Specify the directory that contains the files of the component credentials secrets, under the `manifests` directory. By default, `ccoctl` creates objects in the directory in which the commands are run. To create the objects in a different directory, use the `--output-dir` flag.
+<3> Optional: Specify the directory that contains the credentials data YAML file. By default, `ccoctl` expects this file to be in `<home_directory>/.nutanix/credentials`. To specify a different directory, use the `--credentials-source-filepath` flag.
+
+. Edit the `install-config.yaml` configuration file so that the `credentialsMode` parameter is set to `Manual`.
++
+.Example `install-config.yaml` configuration file
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: cluster1.example.com
+credentialsMode: Manual <1>
+...
+----
+<1> Add this line to set the `credentialsMode` parameter to `Manual`.
+
+. Create the installation manifests by running the following command:
++
+[source,terminal]
+----
+$ openshift-install create manifests --dir <installation_directory><1>
+----
+<1> Specify the path to the directory that contains the `install-config.yaml` file for your cluster.
+
+. Copy the generated credential files to the target manifests directory by running the following command:
++
+[source,terminal]
+----
+$ cp <ccoctl_output_dir>/manifests/*credentials.yaml ./<installation_directory>/manifests
+----
+
+.Verification
+
+* Ensure that the appropriate secrets exist in the `manifests` directory.
```

Comment:
Consider being more specific about which secrets the user should be looking for. (I'm not sure which ones are appropriate.)

Reply:
@jeana-redhat not sure if you can/should be more explicit here.

Reply:
Would have to check with dev on that, and I expect it varies from release to release. There might be a way for them to check it against something else so we don't have to maintain a list and change it as the product changes. Good backlog item (I bet this applies to other platforms as well, historically we have not been super explicit on this type of topic).